scripts/lib/setup-apt-repo

```diff
@@ -47,20 +50,53 @@ deb http://ppa.launchpad.net/groonga/ppa/ubuntu $release main
 deb-src http://ppa.launchpad.net/groonga/ppa/ubuntu $release main
 EOF
 elif [[ "$release" =~ ^(buster)$ ]]; then
+    distribution=debian
     apt-key add "$SCRIPTS_PATH"/setup/pgdg.asc
-    apt-key add "$SCRIPTS_PATH"/setup/pgroonga-debian.asc
     cat >$SOURCES_FILE <<EOF
 deb http://apt.postgresql.org/pub/repos/apt/ $release-pgdg main
 deb-src http://apt.postgresql.org/pub/repos/apt/ $release-pgdg main
-
-deb https://packages.groonga.org/debian/ $release main
-deb-src https://packages.groonga.org/debian/ $release main
 EOF
 else
     echo "Unsupported release $release."
     exit 1
 fi
 
+if ! dpkg-query --list | grep --quiet "^ii  groogna-apt-source"; then
```

Comment:
We work hard to make sure this script is fast in the noop case where things are already up to date; it looks like this takes 40ms, which is probably OK; but we could consider checking for the existence of a file in that package.

Reply:
Oh! I found a typo in the command line:

```text
groogna
groonga
    ^^
```

I'll fix it.

Reply:
> we could consider checking for the existence of a file in that package.

We can use `/etc/apt/sources.list.d/groonga.sources` for the propose.

Reply:
Fixed the typo.

If file existence check approach is better, I'll change this approach.

Reply:
Yeah, I think the file existence check is probably better; we run this script every time we do a version upgrade (via `zulip-puppet-apply`, so we're trying to make its noop case as instant as possible.

Reply:
I note that `groonga-apt-source` doesn’t actually clean up `/etc/apt/sources.list.d/groonga.sources` on removal, only on purge. Either `groonga-apt-source` should be fixed to clean it up on removal, or we should test a file outside of `/etc`.

Reply:
Oh, I didn't notice it.
I changed to use `/usr/share/doc/groonga-apt-source/copyright`.
Can we merge this now?

I'm not sure whether `groonga-apt-source` should remove `/etc/apt/sources.list.d/groonga.sources` on `apt remove` or not. It seems that `apt remove` should not remove files under `/etc/`.